possel/commands.py

```diff
@@ -99,9 +112,17 @@ def dispatch(self, buffer_id, line):
         command, *rest = line.split(maxsplit=1)
         command = command.lower()
 
-        if command in commands:
+        if command in prefix_commands:
             buffer = model.IRCBufferModel.get(id=buffer_id)
-            getattr(self, command)(buffer, rest)
+            try:
+                actual_command, = prefix_commands[command]
```

Comment:
Would probably be clearer to just check the length of prefix_commands and take the first element.